### ISSUE TITLE
Nefunkční doplněk

### ISSUE THREAD

**Issue by JanSykor** [alladdin/plugin.video.primaplay#20]
Po spuštění se objeví pouze "Hledej" a to je vše co zbylo z úvodního menu.
Na prima play  https://www.iprima.cz/#D-PrimaPlay  je hláška:

Videoarchiv Prima PLAY dohrál…
…všechna videa sledujte na www.iprima.cz

Dá se s tím něco dělat?
Díky
Honza

**Comment by S474N** [comment]
Zrovna s tim sem jdu, presne tak, asi to bude chtit zmenit URL.

https://www.mediar.cz/prima-play-konci-archivem-vysilani-bude-iprima/

**Comment by ToKroCZ** [comment]
Ano, změnilo se vše z _play.iprima.cz_ na _prima.iprima.cz_. Co jsem tak zběžně koukal, tak formát URL ale jinak vesměs zůstal, takže fix by snad mohl být docela snadný. @alladdin @lzoubek

**Comment by alladdin** [comment]
Jen tak v rychlosti jsem se na to koukal a zmenila se nejen url ale i struktura toho webu, takze to bude chtit trochu vetsi zasah. Bohuzel na to do vanoc nebudu mit cas.
